functests/4_latency/latency.go

```diff
@@ -349,17 +350,20 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 	return latencies
 }
 
-// initMaximumLatencyValue should look for one of the following environment variables:
+// checkMaximumEnvironLatencyValue should look for one of the following environment variables:
 // OSLAT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // CYCLICTEST_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // HWLATDETECT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
-func initMaximumLatencyValue(testName string) {
+func checkMaximumEnvironLatencyValue(testName string) error {
+	if testName != strings.ToUpper(oslatTestName) && strings.ToUpper(cyclictestTestName) != strings.ToUpper(hwlatdetectTestName) {
```

Comment:
copy/paste artifact?
this part of the `if` seems to be always true:
`strings.ToUpper(cyclictestTestName) != strings.ToUpper(hwlatdetectTestName)`

Reply:
yup. nice catch